libs/payments/currency/src/lib/currency.config.ts

```diff
@@ -13,10 +13,18 @@ export class CurrencyConfig {
   )
   @IsObject()
   public readonly taxIds!: { [key: string]: string };
+
+  @Transform(
+    ({ value }) => (value instanceof Object ? value : JSON.parse(value)),
+    { toClassOnly: true }
+  )
+  @IsObject()
+  public readonly currenciesToCountries!: { [key: string]: string[] };
```

Comment:
Does this need to be a config variable or could it just be a const map? Changes probably would happen, but won't happen too often. Either way is good.

Reply:
It's already an env var in auth server, so I figured we'd want the same here. If that's not true, I'm good with using a const.